HIRS_AttestationCAPortal/src/main/webapp/WEB-INF/jsp/certificate-details.jsp

```diff
@@ -321,6 +318,29 @@
                     </div>
                 </c:when>
                 <c:when test="${param.type=='platform'}">
+                    <c:if test="${not empty initialData.platformType}">
+                        <div class="row">
+                            <div class="col-md-1 col-md-offset-1"><span class="colHeader">Platform Type</span></div>
+                            <div id="platformType" class="col col-md-8">${initialData.platformType}</div>
+                        </div>
+                        <div class="row">
+                            <div class="col-md-1 col-md-offset-1"><span class="colHeader">Platform Chain</span></div>
+                            <div id="platformType" class="col col-md-8">
+                                <span>
+                                    <c:forEach items="${initialData.chainCredentials}" var="credential" varStatus="loop">
```

Comment:
Change this to initialData.chainCertificates if you change the key in CertificateStringMapBuilder.

Reply:
Updated to reflect the change.